### ISSUE TITLE
prefix related issue 

### ISSUE THREAD

**Issue by diwalihabahi** [TheAxiome/discord_auto_prefix#1]
Hey, i am useing your pkg i am facing problem  const client = new Discord.Client()

```const { aPrefix } = require('discord_auto_prefix')
const prefix = new aPrefix()

client.on('ready', () => {
    console.log(`${client.user.tag} is now online!`)
})

client.on('guildCreate', async guild => {
    prefix.defaultPrefix(guild, "!") //Sets the bots default prefix when it joins a new guild
})

client.on('message', async message => {
    const PREFIX = await prefix.fetchPrefix(message) //Fetch's the guilds prefix


    if (message.content === `${PREFIX}ping`) {
        message.channel.send(`PONG! My prefix  is ${PREFIX}`)
    }

    if (message.content.startsWith(`${PREFIX}setprefix`)) {
        const args = message.content.slice(10)

        prefix.setPrefix(message, args) //Sets the guilds prefix
    }```
                                
**after doing this what should i put on start's with because after changeing a guild prefix not able to use my commands what should i put after**  message.content.startsWith(onthisplace)  ```if (message.content.startsWith(prefix)) {
    const args = message.content.slice(prefix.length).trim().split(/ +/);```

**Comment by TheAxiome** [comment]
Thank you for telling me this issue! I will work on a fix immediately. I will reply back to your issue with another comment when the fix has been made!

**Comment by TheAxiome** [comment]
@diwalihabahi Update your version of [discord_auto_prefix](https://www.npmjs.com/package/discord_auto_prefix) to version 1.4.0, it fixes the problem, and a lot of bugs, and adds a new command!

**Comment by TheAxiome** [comment]
This issue has been closed, because it has been fixed, just update to version 1.4.0 of [discord_auto_prefix](https://www.npmjs.com/package/discord_auto_prefix)
